spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/SpanAdjuster.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2013-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.sleuth;
+
+/**
+ * Adds ability to adjust a span before reporting it.
+ *
+ * @author Marcin Grzejszczak
+ * @since 1.1.4
+ */
+public interface SpanAdjuster {
+	/**
+	 * You can adjust the {@link Span} by creating a new one using the {@link Span.SpanBuilder}
+	 * before reporting it. That way you can e.g. change the name of the span.
```

Comment:
make a code example javadoc of the same.

ex s -> span.toBuilder().name(scrub(span.name)).build();